README.md

```diff
@@ -178,6 +178,31 @@ For more information on Ansible collection usage, see [Ansible's official usage
 
 Use-case examples for this collection can be found [here](./examples/README.md).
 
+## Development
+
+The following section outlines various information relating to the development of this collection.
+
+### Attaching a Debugger
+
+To quickly and easily attach a debugger to a running module in this collection, 
+you can use the [madbg](https://pypi.org/project/madbg/) package:
```

Comment:
Is there any alternative debuggers? The development of the [madbg repository](https://github.com/kmaork/madbg) doesn't seem to be active..

Reply:
The Ansible-recommended debugger is [epdb](https://pypi.org/project/epdb/) but it doesn't seem to work with Ansible's stdout weirdness anymore, and isn't being too actively maintained either. It'd definitely be worth looking into other projects though 👀